cla-backend/cla/models/github_models.py

```
@@ -2173,77 +2173,119 @@ def pygithub_graphql(g, query: str, variables: dict | None = None):
+          ... on Commit {
+            oid
+            message
+            author { name email user { databaseId login name email } }
+          }
+        }
+      }
+    }"""
+    head = pygithub_graphql(g, q_head, {"owner": owner, "name": repo_name, "oid": head_oid})["repository"]["object"]
+    if head is None:
+        cla.log.error(f"Failed to fetch head commit for {owner}/{repo_name} PR #{number}")
+        raise ValueError("failed to fetch head commit using GraphQL")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)